yb-voyager/cmd/importData.go

```diff
@@ -194,7 +194,8 @@ func getYBServers() []*tgtdb.Target {
 	}
 
 	if parallelImportJobs == -1 {
-		parallelImportJobs = len(targets)
+		parallelImportJobs = fetchParllelJobs(targets)
+		utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
 	}
```

Comment:
`getYBServers()` is doing two jobs:
- get the list of yb servers
- find out default value of `parallelImportJobs`

Let's move the code associated with the second aspect to a separate function `getDefaultImportJobs()`. And move the call to the new function inside the `importData()`.